### PR TITLE
TW-1164 changing pricing notes for new enterprise plans

### PR DESCRIPTION
**Pull request by jkonrath-postman**

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/administration/about-postman-enterprise.md [hunk 1]
```diff
@@ -1,51 +1,56 @@
...
+
+The Private API Network provides a [central directory](https://www.postman.com/api-platform/api-catalog/) of workspaces, collections, and APIs your team uses internally. Your Postman team can access these resources and start using them right away. By using the Private API Network, you can enable developers across your organization to discover, consume, and track API development in one place.
+
 ### API Governance and API Security features
 
-Postman Enterprise provides you with advanced [API governance](https://www.postman.com/api-platform/api-governance/) and [API security](https://www.postman.com/api-platform/api-security/) features:
+Postman Enterprise plans provide you with advanced [API governance](https://www.postman.com/api-platform/api-governance/) and [API security](https://www.postman.com/api-platform/api-security/) features:
 
-* **Custom token scanner** - Create a [custom token scanner](/docs/administration/token-scanner/#custom-alerts) to proactively check for security leaks.
+* **Custom secret scanner** - Create a [custom secret scanner](/docs/administration/token-scanner/#custom-alerts) to proactively check for security leaks.
```
Comment (claireriley) [inline src/pages/docs/administration/about-postman-enterprise.md:45]:
```suggestion
* **Secret Scanner** - Create [custom alerts](/docs/administration/token-scanner/#custom-alerts) to proactively check for security leaks.
```

**Review by claireriley — APPROVED** [review]
LGTM! One minor note

File: src/pages/docs/administration/billing.md [hunk 2]
```diff
@@ -261,10 +258,19 @@ In your [billing dashboard](http://go.postman.co/billing), you can view:
 * How much you will be billed at the end of your cycle (unless you add or remove users in the interim)
 * Additional information about your Auto-Flex and regular billing cycles
 
-<img alt="Billing overview with auto flex" src="https://assets.postman.com/postman-docs/auto-flex-dash-9.14.jpg" />
```
Comment (taehoshino) [inline src/pages/docs/administration/billing.md:264]: @jkonrath-postman Why did we have to remove this image but not the one in the above "Viewing billing overview" section?

**Pull request merged**